helm/resource_release_test.go

```diff
@@ -1796,3 +1817,18 @@ func TestResourceExampleInstanceStateUpgradeV0(t *testing.T) {
 		}
 	}
 }
+
+func testAccHelmReleaseSetListValues(resource, ns, name string) string {
+	return fmt.Sprintf(`
+resource "helm_release" "%s" {
+	 name        = %q
+	namespace   = %q
+	  chart       = "./testdata/charts/umbrella-chart"
+
+	set_list {
+		name = "test"
```

Comment:
you'll need to add this value to the chart [values.yaml](https://github.com/hashicorp/terraform-provider-helm/blob/main/helm/testdata/charts/test-chart-v2/values.yaml)